docs/organizations/using-a-coding-standard.md

```diff
@@ -94,6 +94,16 @@ To edit an existing coding standard or change the repositories that follow that
 !!! tip
     To ensure that all new repositories automatically follow the coding standard, [set the coding standard as default](#set-default).
 
+## Deleting a coding standard {: id="deleting"}
+
+To delete a coding standard:
+
+1.  Open your organization **Coding standards** page, tab **Coding standards**.
+
+1.  Click the trash can icon on the coding standard card and confirm by clicking **Delete coding standard**.
```

Comment:
Are we OK merging these closely related steps in one instruction?

Reply:
Yes, and I wouldn't even mention the confirmation step as it should be obvious/straightforward for users (in other situations where there's a confirmation prompt our docs don't mention it as well).

Reply:
Thanks, @prcr! I'd still like to leave the "and confirm" fragment at the end of the sentence to differentiate it from a previous step that does not require confirmation (changing the default CS) and to reassure the user that this destructive action does require 2 clicks.

Reply:
```suggestion
1.  Click the trash can icon on the coding standard card and confirm.
```